sdcm/nemesis.py

```diff
@@ -5900,6 +5917,25 @@ def disrupt(self):
         self.call_random_disrupt_method(disrupt_methods=self.disrupt_methods_list, predefined_sequence=True)
 
 
+class TemporaryMonkey(Nemesis):
```

Comment:
not clear what is this class and this comment

Reply:
removed